src/App.js

```diff
@@ -31,20 +32,21 @@ export default function App() {
     });
   }, [])
 
-     // useEffect (() => {
-  //   fetch("http://localhost:3030/trainers")
-  //   .then((res) => res.json())
-  //   .then((trainerData) => {
-  //     setMembers(trainerData);
-  //     console.log("Inside Trainer Get Fetch: ", trainerData)
-  //   });
-  // }, [])
+     useEffect (() => {
+    fetch("http://localhost:3030/trainers")
```

Comment:
This will need to change to use something like `${process.env.BACKEND_URL}/trainers`, and in your .env file you'd have
```
BACKEND_URL="http://localhost:3030"
```